presto-hive-hadoop2/bin/run_hive_tests.sh

```diff
@@ -7,6 +7,11 @@ set -euo pipefail -x
 cleanup_docker_containers
 start_docker_containers
 
+# restart HMS to pickup memory settings
```

Comment:
Can this be added directly to the docker container to avoid this restart?

CC: @aweisberg

Reply:
I think we need to directly bump the memory settings in the docker container